integration-tests/graphql.spec.js

```diff
@@ -0,0 +1,136 @@
+'use strict'
```

Comment:
Why is this an integration test? Integration tests should be reserved for use cases that require a sandboxed tracer, like validating dependencies.

Reply:
Since the GraphQL system tests were not completed by that time we wanted to include an end to end test with apollo server in order to be able to detect possible misalignments on future versions. Maybe I didn't understand quite right what was the intention of the integrations tests. What would be the proper way to add these kind of tests?

Reply:
Apollo should already be available in the existing tests. Plugin tests are basically integration tests for plugins in that they use the real library with traces sent to a mock agent. The integration tests from the `integration-tests` folder are meant to test things that require installing the library through npm.